main.py

```diff
@@ -27,16 +27,28 @@ def multi_invite(org: str, a_users: list):
         uid = gh.get_user_id(user)
 
         if uid is None:
-            cprint(f"[red]: user {user} does not exists")
+            cprint(f"[red]: user {user} does not exist or is already a member of the organization")
```

Comment:
```suggestion
            cprint(f"[red]: user {user} does not exist.")
```
UID can only be None if there is no GitHub account on that username ( no account ) so a member cant exist in organisation if the username doesn't exists